src/classes/Account.js

```diff
@@ -170,17 +173,20 @@ class Account {
 					sub = [sub];
 				}
 
-				// Display premium info if user has both premium and plus subscriptions
+				const subscriptions = [];
+
 				const premiumSubscription = sub.find(subscription => subscription.productName.includes('Ghostery Premium'));
 				if (premiumSubscription) {
 					this._setSubscriptionData(premiumSubscription);
+					subscriptions.push(premiumSubscription);
 				}
 
 				const plusSubscription = sub.find(subscription => subscription.productName.includes('Ghostery Plus'));
 				if (plusSubscription) {
+					subscriptions.push(plusSubscription);
 					this._setSubscriptionData(plusSubscription);
 				}
-				return customer;
+				return subscriptions;
```

Comment:
Looking much better, just a couple more things: 
* We can simplify/dedupe this by usng `forEach` to iterate once over the `sub` array, checking whether the `productName` of each element is in a `productNames` array we define, and pushing the element to the `subscriptions` array if so.
* We need to make sure we don't clobber Premium subsciption data with Plus subscription data in `_setSubscriptionData` in case the user has both and the Plus sub is found after the Premium one.

Reply:
Let's not worry about the `forEach` change for now